commands/@.js

```diff
@@ -17,6 +17,28 @@ function isAdmin(){
   return user.id == admin_id
 }
 
+function canRemoveByAngryPoints(value) {
+  if (value > 0) {
+    return false
+  }
+  var pvalue = -value // convert amount to positive
+  var AngryPointsLimit = Libs.ResourcesLib.userRes("AngryPointsLimit")
```

Comment:
i think it must be
var angryPointsHourlyLimit = Libs.ResourcesLib.userRes("angryPointsHourlyLimit")

Reply:
it is not hourlyy limit.
it is limit of master can remove per request

Reply:
so name like AngryPointsLimitPerRequest  is more good